resources/lang/en/routes.php

```diff
@@ -6,4 +6,8 @@
     'login' => 'login',
     'register' => 'register',
     'verification.verify' => '/verify-email/{id}/{hash}',
+    'projects' => 'projets',
```

Comment:
Because this is the English routes.php file, should this actually be:
```suggestion
    'projects' => 'projects',
```

Reply:
You're right, thanks!